cloudify_cli/cli/helptexts.py

```diff
@@ -242,5 +242,6 @@
 SECRET_VALUE = "The secret's value to be set"
 SECRET_STRING = "The string to use as the secret's value"
 SECRET_FILE = "The secret's file to use its content as value to be set"
+SECRET_UPSERT = 'Update secret with new value if it already exists'
```

Comment:
Do we not mean: "Update secret with new value if it exists, otherwise create it."?

Reply:
(I realise this is on a create command so create is somewhat implicit, but this feels like it might be better explicit)